libnmstate/nm/nmclient.py

```diff
@@ -27,7 +27,6 @@
     gi.require_version('NM', '1.0')  # NOQA: F402
     from gi.repository import NM     # pylint: disable=no-name-in-module
 except ValueError:
-    # Allow the error to pass in case we are running in a unit test context
```

Comment:
The comment is still valid, is it not? Otherwise the detailed error message should happen here.

Reply:
Not exactly. It is not for the unit test, it is for the user who attempts to run it without the proper dependencies installed.
Running stuff from the root on import is nasty, we use this to avoid the noise here and manage it in a more controlled manner.